apps/site/lib/site_web/templates/page/_find_a_location.html.eex

```diff
@@ -0,0 +1,29 @@
+<div class="c-grid-buttons-right">
+  <div class="c-location-header">
+    <h2>
+      Find a Location
+    </h2>
+  </div>
+  <div class="c-location-buttons-container">
+    <%= link to: cms_static_page_path(@conn, "/stops"), class: "c-location-button" do %>
+      <div class="c-grid-button__content">
+        <span class="c-grid-button__icon">
+          <%= svg("stations-parking.svg") %>
+        </span>
+        <span class="c-grid-button__name">
+          Stations and Parking
+        </span>
+      </div>
+    <% end %>
+    <%= link to: cms_static_page_path(@conn, "/transit-near-me"), class: "c-location-button" do %>
+      <div class="c-grid-button__content">
+        <span class="c-grid-button__icon">
+          <i class="fa fa-map-location-dot" aria-hidden=true></i>
+        </span>
```

Comment:
```suggestion
        <%= fa "map-location-dot", class: "c-grid-button__icon" %>
```

I think using our `fa` helper function will work for this.

Reply:
The sm breakpoint matches the mocks on local chrome for me 🤔 
<img width="565" alt="Screen Shot 2022-05-23 at 1 06 40 PM" src="https://user-images.githubusercontent.com/91911166/169873666-ac74c3e3-685b-4296-90ef-62352539fa5a.png">

Reply:
Ahh maybe that's on me then. Well, we can merge that bit as is and we can always update it later if it's off somewhere.